### PR TITLE
Select correct assignment name

### LINKED ISSUE #67: Failed to construct psbt when do tansfer
```
rgb -d .alice -w default transfer rgb:28hM47M-EZ7N4qCDB-gF2Mh73Vx-nJHhgLM7m-2671i3NJY-bLU8TzT/RGB20/1000+tb1pgdl5cks7zu7wymr5vwwcc5rstvkxuj03c8qctvarx5g6qgfsgjms04f86n 100 consignment.rgb
```

Error: field name beneficiary is unknown to the contract interface

### PR DESCRIPTION
**Pull request by crisdut**
Closes #67

### CODE REVIEW

File: src/pay.rs [hunk 1]
```diff
@@ -170,17 +170,20 @@ impl Runtime {
...
             .assignment
             .as_ref()
             .or_else(|| {
-                iface
-                    .transitions
-                    .get(operation)
-                    .and_then(|t| t.default_assignment.as_ref())
+                iface.transitions.get(operation).and_then(|t| {
+                    t.default_assignment
+                        .clone()
```
Comment (dr-orlovsky) [inline src/pay.rs:180]: why do we need clone here?

Comment (crisdut) [reply src/pay.rs:180]: Without clone, i receive the compile error below:

```cannot move out of `t.default_assignment` which is behind a shared reference```

Comment (dr-orlovsky) [reply src/pay.rs:180]: Do `as_ref` instead of clone

Comment (crisdut) [reply src/pay.rs]: Done

**Review by dr-orlovsky — APPROVED** [review]
ACK https://github.com/RGB-WG/rgb/commit/721b55c7167e8d3c1089f91cca160d5645abb9f1

**Pull request merged**